config/locales/en_CA.yml

```diff
@@ -810,7 +810,9 @@ en_CA:
         search: Search
       sort:
         pagination:
-          total_html: "<strong>%{total} products</strong> found for your search criteria. Showing %{from} to %{to}."
+          total_html:
```

Comment:
Whenever a translation needs to be added or updated, we only do the changes in `en.yml`. Other locales are then managed by Transifex. For reference.
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-%28i18n%29#transifex-and-github-integration

Please keep the `en.yml` changes and revert the changes from the other locale's yml file. Thanks

Reply:
Thanks @chahmedejaz 
Now I understand better :bow: